coalib/coala_modes.py

```diff
@@ -4,6 +4,7 @@ def mode_normal(console_printer, log_printer, args, debug=False):
 
     :param console_printer: Object to print messages on the console.
     :param log_printer:     A LogPrinter object to use for logging.
+                            It is now deprecated and unused.
```

Comment:
Could you maybe file a new issue about deprecating this parameter? This would also include emitting a warning and not only a doc change ;)

Reply:
Filed issue #4972